docs/fxa-engineering/fxa-sub-platform.md

```diff
@@ -169,13 +169,120 @@ server:
 The payments server handles the payment flow as well as serving pages for managing
 a user's subscription that are linked from the Settings page.
 
+[subscription endpoints]: https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#subscriptions
+
 ### Auth Server
 
 FxA's Auth Server makes Stripe API calls for authenticated FxA users via its [subscription
 endpoints][]. Stripe updates are sent back to the Auth Server via Stripe webhooks when a
 users subscription has been created/updated/deleted.
 
-[subscription endpoints]: https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/docs/api.md#subscriptions
+Some Stripe webhooks will trigger emails.  These emails are behind a feature flag.  If you wish to send emails in your environment, set the auth server configuration
+
+```
+{
+  "subscriptions": {
+    "transactionalEmails": {
+      "enabled": true
+    }
+  }
+}
+```
+
+or the environment variable `SUBSCRIPTIONS_TRANSACTIONAL_EMAILS_ENABLED` to "true".  In order to receive Stripe webhook events in your local development, you need to use the [Stripe CLI](https://stripe.com/docs/stripe-cli/webhooks)'s event forwarding feature.
```

Comment:
Should we mention that the emails can then be seen in the inbox service when running locally?

Reply:
I can link to the relevant README.md section in FxA.